### PR TITLE
Update CSS

### PR DESCRIPTION
**Pull request by jswale**
Update Stories & too much

👊 First pass for the review 👊

### CODE REVIEW

File: src/components/atoms/p6-hint/p6-hint.stories.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  Prop,
+} from "../../../shared/storybook/stories";
+import { Mode } from "../../../shared/types";
+
+const getHint = (text: string, props?: Prop): string => {
+  return getComponent("p6-hint", text, props);
+};
+
+/*
+type DefaultStoryArgs = {
```
Comment (vuongc) [inline src/components/atoms/p6-hint/p6-hint.stories.ts:16]: shouldn't you use this exemple now ?

File: src/components/atoms/p6-switch/p6-switch.tsx [hunk 2]
```diff
@@ -16,19 +39,68 @@ export class P6Switch {
...
+  /**
+   * Fires when the field has been checked for validity and doesn't satisfy its constraints
+   */
+  @Event() p6Invalid!: EventEmitter<InvalidEventDetail>;
+
+  @State() hasError = false;
+
+  private nativeInput: HTMLInputElement | undefined;
+
+  // private defaultValue: boolean = this.checked;
```
Comment (vuongc) [inline src/components/atoms/p6-switch/p6-switch.tsx:81]: should remove this comment

File: src/components/atoms/p6-switch/p6-switch.tsx [hunk 3]
```diff
@@ -16,19 +39,68 @@ export class P6Switch {
...
+
+  componentWillLoad(): void {
+    // this.defaultValue = this.checked;
```
Comment (vuongc) [inline src/components/atoms/p6-switch/p6-switch.tsx:84]: this one too

File: src/components/atoms/p6-switch/p6-switch.tsx [hunk 4]
```diff
@@ -41,4 +113,50 @@ export class P6Switch {
...
+    this.p6FormRegister.emit(this);
+  }
+
+  disconnectedCallback?(): void {
+    this.p6FormUnregister.emit(this);
+  }
+
+  /*
+  @Watch("checked")
+  onCheckedUpdated(newValue: boolean) {
```
Comment (vuongc) [inline src/components/atoms/p6-switch/p6-switch.tsx:127]: remove this one too

File: src/global/base.scss [hunk 5]
```diff
@@ -1,2 +1,152 @@
...
+  font-family: $body-family;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+/*
```
Comment (vuongc) [inline src/global/base.scss:29]: Should this be removed ?

Comment (jswale) [reply src/global/base.scss:29]: Not yet. Should be used in a new JIRA for the buttons component

**Pull request merged**